examples/Propagator.ipynb

```diff
@@ -281,6 +281,123 @@
+      "\n",
+      "Stochastic interaction with a size of 467166 MeV at an initial particle energy of 2691671 MeV. The ID of the Interaction Type is epair. The hash of the target that we interacted with is 9764564051507188568.\n",
+      "\n",
+      "Stochastic interaction with a size of 331392 MeV at an initial particle energy of 1875243 MeV. The ID of the Interaction Type is brems. The hash of the target that we interacted with is 9764564051507188568.\n",
+      "\n",
+      "Stochastic interaction with a size of 341296 MeV at an initial particle energy of 1394992 MeV. The ID of the Interaction Type is brems. The hash of the target that we interacted with is 16056410032740017603.\n",
+      "\n",
+      "Stochastic interaction with a size of 38883 MeV at an initial particle energy of 633084 MeV. The ID of the Interaction Type is photonuclear. The hash of the target that we interacted with is 9764564051507188568.\n",
+      "\n",
+      "Stochastic interaction with a size of 55354 MeV at an initial particle energy of 489354 MeV. The ID of the Interaction Type is brems. The hash of the target that we interacted with is 9764564051507188568.\n",
+      "\n",
+      "Stochastic interaction with a size of 113388 MeV at an initial particle energy of 327325 MeV. The ID of the Interaction Type is brems. The hash of the target that we interacted with is 9764564051507188568.\n",
+      "\n",
+      "Stochastic interaction with a size of 21260 MeV at an initial particle energy of 172330 MeV. The ID of the Interaction Type is brems. The hash of the target that we interacted with is 9764564051507188568.\n",
+      "\n",
+      "Stochastic interaction with a size of 10514 MeV at an initial particle energy of 104390 MeV. The ID of the Interaction Type is ioniz. The hash of the target that we interacted with is 12919284898413356206.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"--- List of stochastic interactions ---\\n\")\n",
+    "for loss in track.stochastic_losses():\n",
+    "    print(f\"Stochastic interaction with a size of {loss.energy:.0f} MeV\", \n",
+    "          f\"at an initial particle energy of {loss.parent_particle_energy:.0f} MeV.\",\n",
+    "          f\"The ID of the Interaction Type is {pp.particle.Interaction_Type(loss.type).name}. The hash of the target that we interacted with is {loss.target_hash}.\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that our `StochasticLoss` objects carries the information about the target that we interacted with in the form of a hash.\n",
+    "For Ionization interactions, this corresponds to the hash of a Medium object.\n",
+    "For all other interactions, this corresponds to the hash of a Component object (i.e. an atom).\n",
+    "We can get the Medium/Component object corresponding to these hashes using the methods `pp.medium.medium_map()` and `pp.component.component_map()`:"
```

Comment:
Due to the recent changes, this comment should refer to the methods `pp.medium.get_medium_for_hash()` and `pp.component.get_component_for_hash()`